### ISSUE TITLE
Promote action

### ISSUE THREAD

**Issue by ad-m** [ad-m/github-push-action#11]
We can add our Github Action on https://github.com/boyney123/github-actions after merging #10.

**Comment by tenhobi** [comment]
Also, we can add it to https://github.com/sdras/awesome-actions

**Comment by tenhobi** [comment]
After #14 and #10 we should make a new release.

**Comment by ad-m** [comment]
@tenhobi , draft new release is available at https://github.com/ad-m/github-push-action/releases/tag/untagged-f5d715bf1918e7ec2d25 .

**Comment by ad-m** [comment]
Release published.

**Comment by ad-m** [comment]
PR created: https://github.com/boyney123/github-actions/pull/57

**Comment by tenhobi** [comment]
Cool!

**Comment by ad-m** [comment]
Another PR created: https://github.com/sdras/awesome-actions/pull/240

Comments are welcome in the case of a delay in accepting changes.
